packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dashboard.html

```diff
@@ -55,12 +55,24 @@
                     <div class="oui-tile__definition d-flex min-w-0">
                         <span class="oui-icon mr-3 oui-icon-user"></span>
                         <div class="word-break min-w-0">
-                            <div class="mb-4">
+                            <div
+                                data-ng-if=":: $ctrl.user.auth.method !== 'provider'"
+                                class="mb-4"
+                            >
                                 <strong
                                     data-ng-bind=":: $ctrl.user.firstname + ' ' + $ctrl.user.name"
                                 ></strong>
                                 <p data-ng-bind="::$ctrl.user.email"></p>
                             </div>
+                            <div
+                                data-ng-if=":: $ctrl.user.auth.method === 'provider'"
```

Comment:
related to this :
https://github.com/ovh/manager/pull/6406/files?file-filters%5B%5D=.html&file-filters%5B%5D=.json&show-viewed-files=true#r798560747